tests/suite/sapi/tests/cart-reorder/SUITE-SAPI15-cart-reorder_50.js

```diff
@@ -0,0 +1,46 @@
+import { loadDefaultOptions } from '../../../../../lib/utils.js';
+import {
+    SharedCartReorderScenario
+} from '../../../../cross-product/sapi/scenarios/cart-reorder/shared-cart-reorder-scenario.js';
+import { SharedCheckoutScenario } from '../../../../cross-product/sapi/scenarios/checkout/shared-checkout-scenario.js';
+export { handleSummary } from '../../../../../helpers/summary-helper.js';
+
+const vus = 10;
+const iterations = 1;
+
+const environment = 'SUITE';
+const thresholdTag = 'cart_reorder_50';
+
+const sharedCheckoutScenario = new SharedCheckoutScenario(environment);
+const sharedCartReorderScenario = new SharedCartReorderScenario(environment);
+
+export const options = loadDefaultOptions();
+options.scenarios = {
+    SAPI15_cart_reorder_50: {
+        exec: 'execute',
+        executor: 'per-vu-iterations',
+        tags: {
+            testId: 'SAPI15',
+            testGroup: 'Cart Reorder',
+        },
+        vus: vus,
+        iterations: iterations,
+    },
+};
+options.thresholds[`http_req_duration{name:${thresholdTag}}`] = ['avg<300'];
+
+export function setup() {
+    return sharedCheckoutScenario.dynamicFixturesHelper.haveCustomersWithQuotes(vus, iterations, 50);
+}
+
+export function execute(data) {
+    const customerIndex = (__VU - 1) % data.length;
+    const { customerEmail, quoteIds } = data[customerIndex];
+    const quoteIndex = __ITER % quoteIds.length;
+
+    // Place an order
+    const checkoutResponseJson = sharedCheckoutScenario.haveOrder(customerEmail, quoteIds[quoteIndex], false);
```

Comment:
It is repeated everywhere, does it make sense to move to a helper?

Reply:
If it would be a helper in AbstractScenario, this class will be a rich model with N helpers, I would avoid it. Instead of helper, we could reuse scenario methods.